ui/selectors/selectors.js

```diff
@@ -851,7 +851,7 @@ export const getFullTxData = createDeepEqualSelector(
     return getTransaction(state, transactionId);
   },
   (_state, _transactionId, _status, customTxParamsData) => customTxParamsData,
-  (txData, transaction, _status, customTxParamsData) => {
```

Comment:
this looks right to me when I step through the code while running the example. 

I think I was (and maybe still am a bit confused by `input` vs. `output` selectors that are passed to the `createSelector` function as described in the [reduxToolkit docs](https://redux.js.org/usage/deriving-data-selectors#createselector-overview). 

@brad-decker are you able to elucidate this pattern a bit better for me?

Reply:
Is the last arg passed always the output selector? And then you pass as many input selector before that as there are args? That doesn't seem to match what we have here... I think I'm overcomplicating this.

Reply:
One second while I look up what 'elucidate' means